src/components/AddBook.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+const AddBook = () => {
+  const categories = [
+    {
+      id: 1,
+      name: 'Action',
+    },
+    {
+      id: 2,
+      name: 'Comedy',
+    },
+    {
+      id: 3,
+      name: 'Drama',
+    },
+    {
+      id: 4,
+      name: 'Triller',
+    },
+  ];
+
+  //   const [category, setCategory] = useState('');
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.